crmsh/bootstrap.py

```diff
@@ -45,6 +45,9 @@
 COROSYNC_CONF_ORIG = tmpfiles.create()[1]
 SERVICES_STOP_LIST = ["corosync-qdevice.service", "corosync.service", "hawk.service"]
 USER_LIST = ["root", "hacluster"]
+QDEVICE_RELOAD = 0
```

Comment:
This looks really ugly @liangxin1300 ,
Better use something like (in fact the best would be to use `enum` but as it is not native in python2 it would bring compatibility issues):
```
QDEVICE_STATES = {
    'reload': 0,
    'restart': 1,
    'restart_later': 2
}
```

Reply:
@arbulu89 Using `enum` is OK, since qdevice only exist on sle15

Reply:
Changed as using `enum`